internal/dockerignore/ignore.go

```diff
@@ -7,14 +7,14 @@ import (
 	"strings"
 
 	"github.com/docker/docker/builder/dockerignore"
-	"github.com/docker/docker/pkg/fileutils"
+	tiltDockerignore "github.com/tilt-dev/dockerignore"
```

Comment:
How would you feel about calling this something like tilt-fileutils or tilt-docker-fileutils? I was a bit thrown that this wasn't replacing github.com/docker/docker/builder/dockerignore.

Reply:
ya, i think you're right that we expect this to replace builder/dockerignore! But I want to go in the opposite direction! :)

move code from builder/dockerignore to tilt-dev/dockerignore, and add apis for making it easier to log where the dockerignore patterns came from

The 'fileutils' name is vestigial from when this was for things like CopyFile, but that's not really what we want it for
https://pkg.go.dev/github.com/docker/docker/pkg/fileutils?tab=doc